pkg/scripts/cleanup.sh

```diff
@@ -9,8 +9,10 @@
 # -h: help
 set -e
 
+source './pkg/scripts/common.sh'
```

Comment:
Similar logic of `generate-variables.sh`, we can have:

```
if [ "${0##*/}" == "generate-variables.sh" ]; then
  # the script ran directly from terminal
  ROOT_PATH=$(cd $(dirname "${0}")/../..; pwd)
else
  # the script called by install.sh or uninstall.sh
  ROOT_PATH=$(cd $(dirname "${0}"); pwd)
fi
SCRIPT_PATH="${ROOT_PATH}/pkg/scripts"

source "${SCRIPT_PATH}/common.sh"
```

and remove line #40, #46, and #61-65, and edit line $4 and #27 to remove `[-r <root_repo>]`

Obviousely we shouldn't pass ROOT_PATH to this script from `install.sh` and `uninstall.sh` in this case.